explainaboard/metrics/auxiliary/sql_evaluation/exec_eval.py

```diff
@@ -0,0 +1,268 @@
+import asyncio
```

Comment:
It looks these files are borrowed from somewhere we don't recognize, but there are no explanation about borrowing. Please apply following changes:

- Move `explainaboard/metrics/auxiliary` to `explainaboard/contrib` to indicate this is a third-party.
- Check if the original license is compatible with the MIT license. If so, add the original licensing notes to this folder. If not, we can not borrow the code from such source.